test_runner/src/main/kotlin/ftl/reports/xml/model/JUnitTestSuite.kt

```diff
@@ -71,4 +71,59 @@ data class JUnitTestSuite(
 
         return this
     }
+
+    fun mergeTestTimes(other: JUnitTestSuite): JUnitTestSuite {
+        if (this.name != other.name) throw RuntimeException("Attempted to merge ${other.name} into ${this.name}")
+
+        // for each new JUnitTestCase
+        //   iif it failed then pull timing info from old
```

Comment:
typo here.

Reply:
`iif` = if and only if

https://math.stackexchange.com/questions/68293/what-is-the-difference-between-only-if-and-iff

Reply:
i'll fix 😭

Reply:
Fixed.